systemvm/debian/opt/cloud/bin/cs/CsFile.py

```diff
@@ -175,6 +175,6 @@ def deleteLine(self, search):
         self.new_config = list(temp_config)
 
     def compare(self, o):
-        result = (isinstance(o, self.__class__) and set(self.config) == set(o.config))
+        result = (isinstance(o, self.__class__) and self.config == o.config)
```

Comment:
So this means that we no longer compare the set to be equal but the line by line items (in order). Is that really what we want @weizhouapache ?

Reply:
yes, we should compare lines by lines.

the method is being used to compare the configurations for
- haproxy
- conntrackd
- radvd

for all 3, I think the order of configurations in the files matters.